src/main/java/org/skriptlang/skript/lang/experiment/Feature.java

```diff
@@ -0,0 +1,85 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.lang.experiment;
+
+import ch.njol.skript.SkriptAddon;
+import ch.njol.skript.patterns.PatternCompiler;
+import ch.njol.skript.patterns.SkriptPattern;
+
+/**
+ * Experimental feature toggles as provided by Skript itself.
+ *
+ * @author moderocky
```

Comment:
```suggestion
```

and everywhere else

Reply:
:rolling_eyes: Spoiling all my fun here sovde

Reply:
everywhere else too kenzie

Reply:
I can't find that in the suggestion sorry